edge-agent/src/Microsoft.Azure.Devices.Edge.Agent.Core/IModule.cs

```diff
@@ -81,6 +81,16 @@ public enum RestartPolicy
         Unknown
     }
 
+    [JsonConverter(typeof(StringEnumConverter))]
+    public enum PullPolicy
+    {
+        [EnumMember(Value = "always")]
+        Always = 0,
```

Comment:
Emm, we need to think about the naming here.. Always might mean something else (like try to pull every time.. )
Wonder if we should call it Default or something like that.

Reply:
Kubernetes calls it `IfNotPresent`

Reply:
Let me know what you guys prefer.

Reply:
Yeah, I like IfNotPresent.. let's go with that.

Reply:
Talked with Varun offline and he agrees with `IfNotPresent`

Reply:
Sounds good, will rename